index.html

```diff
@@ -90,10 +90,22 @@ <h3>
       <dfn>RTCAudioReceiverStats</dfn> dictionary non-standardized members
     </h3>
     <pre class="idl">partial dictionary RTCAudioReceiverStats {
+      unsigned long long delayedPacketOutageSamples;
       unsigned long long jitterBufferFlushes;
 };</pre>
     <dl data-link-for="RTCAudioReceiverStats" data-dfn-for="RTCAudioReceiverStats" class=
     "dictionary-members">
+      <dt>
+        <dfn><code>delayedPacketOutageSamples</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long-long">unsigned long long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Experimental stat which is available <a href="https://docs.google.com/document/d/1stYIZhEmDZ7NJF9gjjsM66eLFJUdc-14a3QutrFbIwI/edit#" title="experiment description">under origin trial in Chromium starting from M72 version</a>.
+          Measures the number of audio samples that were concealed due to jitter buffer underrun caused by delayed packets.
+          This gives an indication of how well the jitter buffer's network adaptation is working.
```

Comment:
This is a subset of https://w3c.github.io/webrtc-stats/webrtc-stats.html#dom-rtcaudioreceiverstats-concealedsamples - is it identical to that stat?

If it's identical, just document it as "oops, we implemented the stat under two names by accident".

Reply:
How is this different from concealedSamples and if it is, how can you tell if it is due to delayed packets or due to something else? Why is this needed when we have concealedSamples?